docs/vocabulary.md

```diff
@@ -388,6 +389,222 @@ pipeline in corresponding documentation.
+* Aliases: N/A
+* Inputs: Packaged Artifacts
+* Outputs: Signed Artifacts, Signing Record(s)
+* Other Results and Side Effects: N/A
+
+#### Policy
+* Semantics: Verifies that policies are followed, for example:  software is from a trusted source, source repositories are configured correctly, Kubernetes manifests are configured securely, dependencies are signed, code standards are followed, code reviews are completed, there is a secure chain of custody, or appropriate work items or change requests are associated with the change.
+* Aliases: Check, Gate, Provenance
+* Inputs: Policies, Outputs of other steps
+* Outputs: Policy and/or Provenance Reports
+* Other Results and Side Effects: Side effects could include blocking a PR/commit so it can't be merged, discarding or approving an artifact/image.
+
+#### Publish
+* Semantics: Upload software artifacts and documentation to another repository. May also update catalogs, mirrors, release notes, etc.
+* Aliases: Push, Upload, Release
+* Inputs: Packaged (Signed) Artifacts
+* Outputs: Repository URLs
+* Other Results and Side Effects: Repositories are updated.
+
+#### Provision
+* Semantics: Request that a new physical or virtual server, network, or other resource be allocated or created.  Examples include a test cluster or object storage.
+* Aliases: Obtain, Request, Allocate
+* Inputs: Resource Request Parameters
+* Outputs: Provisioned Resources (with connection info)
+* Other Results and Side Effects: Resources are provisioned.
+
+#### Deploy
+* Semantics: Make changes to any environment other than the pipeline environment. Configure the environment. Deploy dependencies, software artificts and/or documentation.
+* Aliases: Install, Configure
+* Inputs: Software Source, Binary Source, Generated Software, Generated Binaries, Dependency List/Graph, Packaged Artifacts, Provisioned Resources
+* Outputs: Routes to Deployments (with connection info). Deployment Records. Secrets to access deployed resources.
+* Other Results and Side Effects: Software running in another environment. Documentation hosted in another environment. 
+
+#### Verify Deployment
+* Semantics: Verify successful deployment of software or documentation.
+* Aliases: Smoke Test
+* Inputs: Provisioned Resources, Routes to Deployments, Secrets to access deployed resources.
+* Outputs: Deployment Verification Results, Deployment Verification Records
+* Other Results and Side Effects: N/A
+
+#### Analyze
+* Semantics: Perform additional processing and analytics based on the results of a previous activity.
+* Aliases: Metrics, Score, Grade, Parse
+* Inputs: Any
+* Outputs: Analysis Results, Analysis Reports
+* Other Results and Side Effects: N/A
+
+#### Message
+* Semantics: Send a message to another system; for example, a Slack message or an Email. This is different from "Create Request" because the pipeline doesn't need to store a link/handle for the new request.
```

Comment:
```suggestion
* Semantics: Send a message to another system; for example, a Slack message or an Email. This is different from "Create Request" because the pipeline doesn't need to store a link/handle for the message.
```
I guess we are talking about the message here? Right?